pkg/tasks/clustermonitoringoperator.go

```diff
@@ -53,11 +63,5 @@ func (t *ClusterMonitoringOperatorTask) Run() error {
 		return errors.Wrap(err, "reconciling Cluster Monitoring Operator ServiceMonitor failed")
 	}
 
-	cr, err := t.factory.ClusterMonitoringClusterRole()
-	if err != nil {
-		return errors.Wrap(err, "initializing cluster-monitoring ClusterRole failed")
-	}
-
-	err = t.client.CreateOrUpdateClusterRole(cr)
-	return errors.Wrap(err, "reconciling cluster-monitoring ClusterRole failed")
+	return nil
```

Comment:
for consistency, this should not return nil, but we should return an `errors.Wrap` above on https://github.com/openshift/cluster-monitoring-operator/blob/43a0a62c2733737fc39634ac3c507d9f46d26742/pkg/tasks/clustermonitoringoperator.go#L62, which itself checks for nil.

Reply:
yes, i am not a fan of the nil swallowing of `errors.Wrap`, but I'll revert for consistency.

Reply:
ack, I get it but we should be consistent so reading the codebase is sane :/

Reply:
yep yep agreed :+1: